setup/src/setup-haskell.ts

```diff
@@ -22,6 +22,34 @@ async function cabalConfig(): Promise<string> {
 export default async function run(
   inputs: Record<string, string>
 ): Promise<void> {
+  core.info(
+    `***************************************************************************`
+  );
+  core.info(
+    `**                                                                       **`
+  );
+  core.info(
+    `**              This action is DEPRECATED.                               **`
+  );
+  core.info(
+    `**                                                                       **`
+  );
+  core.info(
+    `**              Please use haskell-actions/setup instead.                **`
+  );
+  core.info(
+    `**                                                                       **`
+  );
+  core.info(
+    `**              (Note the dash instead of the slash.)                    **`
+  );
+  core.info(
+    `**                                                                       **`
+  );
+  core.info(
+    `***************************************************************************`
+  );
+
```

Comment:
(sometimes the code formatter can be silly...)

This currently shows only when one opens the `setup` group in a run:
<img width="593" alt="Screenshot 2023-09-10 at 09 32 32" src="https://github.com/haskell/actions/assets/1155218/e6595ccc-8031-4a52-b071-50b148c78517">
https://github.com/haskell/actions/actions/runs/6131130563/job/16640832541#step:3:8